### DISCUSSION TITLE
Vercel's Speed Insights uses wrong offensive India Map

### DISCUSSION THREAD

**Discussion by soubhagya1999** [vercel/community#6993] [Bug Report] [answered]
### Page to Investigate

https://vercel.com/soubhagya1999s-projects/soubhagya-portfolio/speed-insights

### Steps to Reproduce

Go to speed insights, make sure **Production** & **Desktop** is selected. Check the **India's Map** in the Countries section.
 **![Screenshot (38)vercel](https://github.com/vercel/community/assets/54418536/41376dc2-ab44-4db8-b760-c61ef2c32e7f)**

**Comment by swarnava** [comment, marked as answer]
Hi,

I have compared with an [official political map of India](https://www.surveyofindia.gov.in/pages/political-map-of-india) ([file](https://www.surveyofindia.gov.in/webroot/UserFiles/files/POL_MAP_4M_ENGLISH_2024.pdf) in English), which I hope to be a reliable source. All the provinces are represented and coloured.

For reference, our map data comes from [NaturalEarth data](https://www.naturalearthdata.com/), which is the most recent, and opensource geographical data that we could find. It’s boadly used in the industry.

**Reply by soubhagya1999** [reply]
Hi,

So what's the conclusion you found after comparing with official political map of India? Did you saw the big difference? Did you saw that the Vercel's Insights India map currently shown does not include the regions of Jammu and Kashmir, and Ladakh, POK which are officially claimed by India, whereas you can find it visible within Indian borders in official political map of India. PFA the screenshots of both the maps.
I hope you understand what I meant, and do the necessary changes to reflect those regions in India's map.

BTW, I visited this link: https://www.naturalearthdata.com/ It's not working, saying "This site can't be reached." So I'm not sure how you guarantee that it's broadly used in the industry.

![image](https://github.com/vercel/community/assets/54418536/bce585ac-5bb6-4759-852c-2f4ef4ce8655)
![image](https://github.com/vercel/community/assets/54418536/4558c51d-6fa8-4ea2-8123-ea1e6887f389)

**Reply by swarnava** [reply]
> BTW, I visited this link: https://www.naturalearthdata.com/ It's not working, saying "This site can't be reached." So I'm not sure how you guarantee that it's broadly used in the industry.

This might be temporary problem with your DNS. Here is how it should look like:


<img width="1279" alt="Screenshot 2024-06-19 at 2 47 49 PM" src="https://github.com/vercel/community/assets/1162991/074f6715-ddea-4863-aff6-9ae95231b67d">

[a comment by an automated account is not shown]
